IdokladSdk/Models/Common/TaxDocumentItemPrices.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace IdokladSdk.Models.Common
+{
+    /// <summary>
+    /// Tax document item prices.
+    /// </summary>
+    public class TaxDocumentItemPrices
+    {
+        /// <summary>
+        /// Gets or sets TotalVat.
```

Comment:
Asi se s tím teď neotravuj, protože ty naše dokumentační stringy nejsou dokonalé, ale místo "Gets or sets Total Vat" bych to napsal "Gets or sets total VAT".